ZeldaFullEditor/Data/Palettes.cs

```diff
@@ -330,7 +354,8 @@ public static bool SavePalettesToROM(byte[] romData)
             }
 
             // TODO: Set to 147 and not 160 because it seems the area numbers for all areas after 147 are messed up and it causes the palettes to save over eachother.
-            WritePalette(romData, Constants.customAreaSpecificBGPalette, OverworldBackgroundPalette, 147);
+            // TODO: Figure out what past me was smoking because this does not mess anything up. It just saves the whole table instead of the last 13 values being left as 0x0000.
```

Comment:
A little confused here. It saves the whole table and that "does not mess anything up" so what's the issue/thing needed to investigate?

Reply:
fair, removed